core/taxonomy.c

```diff
@@ -47,3 +48,31 @@ int taxonomy_index_for_category(struct taxonomy_data *t, enum taxonomy_category
 			return i;
 	return -1;
 }
+
+const char *taxonomy_get_country(struct taxonomy_data *t)
+{
+	for (int i = 0; i < t->nr; i++)
+		if (t->category[i].category == TC_COUNTRY)
+			return t->category[i].value;
```

Comment:
[1] i was wondering about this before. in that case do we want to wrap the `for` loop with `{ }`, given it wraps more than a single line or is the above just fine.

Reply:
From a C/C++ syntax the code as is works - but adding { } wouldn't hurt. We do have this pattern quite a few times in the code, though.